website/docs/developer/deployment-guide/deployment-upgrading-orca.md

```diff
@@ -75,7 +75,11 @@ To update your ORCA version:
 1.	Find the desired release on the [ORCA Releases page](https://github.com/nasa/cumulus-orca/releases).
 
 2.	Update the `source` in your Terraform deployment files for ORCA by
-replacing `vx.x.x` with the desired version of ORCA.
+replacing `vx.x.x` with the desired version of ORCA. If upgrading from `v8.x.x` to `v9.x.x` follow the below steps:
+    1. Run the Lambda deletion script found in `bin/delete_lambda.py` this will delete all of the ORCA Lambdas with a provided prefix. Or delete them manually in the AWS console.
+    2. Navigate to the AWS console and search for the Cumulus RDS security group.
+    3. Remove the inbound rule with the source of `PREFIX-vpc-ingress-all-egress` in Cumulus RDS security group.
+    4. Search for `PREFIX-vpc-ingress-all-egress` and delete the security group **NOTE:** Due to the Lambdas using ENIs, when deleting the securty groups it may say they are still associated with a Lambda that was deleted by the script. AWS may need a few minutes to refresh to fully disassociate the ENIs completely, if this error appears wait a few minutes and then try again.
 
```

Comment:
We should also add this under the migration notes in CHANGELOG.md in the correct release version.
https://github.com/nasa/cumulus-orca/blob/master/CHANGELOG.md#migration-notes-1
https://github.com/nasa/cumulus-orca/blob/master/CHANGELOG.md#800-2023-05-24